api/controllers/reviewController.js

```diff
@@ -0,0 +1,36 @@
+const { reviewService } = require("../services");
+const { catchAsync } = require("../utils/error");
+
+const createReview = catchAsync(async (req, res) => {
+  const { rating, comment, movieId } = req.body;
+  const accessToken = req.headers.authorization;
+  await reviewService.createReview(accessToken, rating, comment, movieId);
+  return res.status(200).json({ message: "CREATE REVIEW SUCCESS" });
+});
+
+const getReview = catchAsync(async (req, res) => {
+  const { movieId } = req.body;
+  const result = await reviewService.getReview(movieId);
+  return res.status(200).json({ result });
+});
+
+const updateReview = catchAsync(async (req, res) => {
+  const { comment, movieId } = req.body;
+  const accessToken = req.headers.authorization;
+  await reviewService.updateReview(comment, movieId, accessToken);
+  return res.status(200).json({ message: "UPDATE REVIEW SUCCESS" });
+});
+
+const deleteReview = catchAsync(async (req, res) => {
+  const { movieId } = req.body;
+  const accessToken = req.headers.authorization;
+  await reviewService.deleteReview(movieId, accessToken);
+  return res.status(200).json({ message: "DELETE REVIEW SUCCESS" });
```

Comment:
```suggestion
  return res.status(204).send();
```
요청에 대한 처리에 맞게 각 응답코드를 적절히 작성해주세요.